README.md

```diff
@@ -465,6 +467,37 @@ extensions such as `import com.klaviyo.pushFcm.KlaviyoRemoteMessage.body` to acc
 We also provide an `Intent.appendKlaviyoExtras(RemoteMessage)` extension method, which attaches the data to your
 notification intent that the Klaviyo SDK requires in order to track opens when you call `Klaviyo.handlePush(intent)`.
 
+## In App Forms
+
+### Prerequisites
+- A published "in-app" form in the Klaviyo forms portal
+- Using version 3.2.0 or higher
+
+### Setup
+The Klaviyo Push SDK for Android works as an extension of the Klaviyo object, and can be chained with your initialization code to allow for form display.
+```kotlin
+Klaviyo
+    .initialize("KLAVIYO_PUBLIC_API_KEY", applicationContext)
+    .registerForInAppForms()
+```
+You can also call it publicly anywhere you can access the Klaviyo forms module:
+```kotlin
+Klaviyo.registerForInAppForms() // make sure you only call this after initializing
+```
+
+
+Once you call the `registerForInAppForms` function, the web view will persist in the background until a form is ready to be shown, or a timeout occurs.
+Consider how often you want to check form forms when you call this, for example:
+* `Activity.onCreate()` -> If you only want forms to appear over a specific activity, checks every time this activity is created.
+* `Application.onResume()` -> Anytime the app is foregrounded, check for forms and show if available.
+* `Activity.onDestroy()` -> If you need the forms to display only once an activity is removed from the view (ex. splash screen).
+
+### Behavior
+
+The web view will listen for forms to display with the same timeout as other requests (10 seconds).
+If no form is available to show, the web view will be removed from memory and there will need to be another call to `registerForInAppForms`
```

Comment:
Can you clarify this a little bit? It sort of reads as a contradiction to the sentence you had above: `Once you call the `registerForInAppForms` function, the web view will persist in the background until a form is ready to be shown, or a timeout occurs. `

Reply:
Actually re-reading everything I had overlooked part of that sentence. "Contradiction" is harsh, but maybe we could better summarize the behaviour of `registerForInAppForms` by leaving out notes on behaviour in the `Setup` section and letting this `Behaviour` section talk to it in one place?